content/docs/apps/production-ready.md

```diff
@@ -83,3 +83,13 @@ When you need a domain name dedicated for your application, DNS delegation for t
 
 #### How
 * See [custom domains]({{< relref "custom-domains.md" >}}).
+
+### Graceful shutdown
+Your application should respond to [shutdown
+events](http://docs.cloudfoundry.org/devguide/deploy-apps/app-lifecycle.html#shutdown)
+by closing connections and cleaning up resources.
```

Comment:
Any chance you have examples? I'm not sure that I've seen a web app that actually needs to do this.